Externals/LZO/lzodefs.h

```diff
@@ -692,7 +692,7 @@
 #  else
 #    define LZO_INFO_ARCH           "arm"
 #  endif
-#elif defined(__arm__) || defined(_M_ARM)
+#elif defined(__arm__) || defined(_M_ARM) || defined(_M_ARM64)
```

Comment:
What about updating the bundled LZO files instead? The current version seems to support arm64.

Reply:
Agree...wow, latest release is 2 years old, and the one in Externals is 9 years old :)